### DISCUSSION TITLE
请问点击收起展开的菜单的时候明显的延时是什么问题导致的

### DISCUSSION THREAD

**Discussion by Ryan-lixuan** [ant-design/ant-design#42710] [Q&A]
当我点击父级菜单想收起展开的子菜单的时候，发现，子菜单先消失，整个变得空白，然后大概延时1秒之后才会收起来，请问这是哪里出了问题，该怎么解决。
![image](https://github.com/ant-design/ant-design/assets/32061860/a963077a-6941-4f73-bcf4-a789bb4be691)

**Comment by Wxh16144** [comment]
是不是动画的原因 🤔

**Reply by Ryan-lixuan** [reply]
他有个关闭延时我已经设为0了，没有任何效果，你们没有出现这个问题吗，我新搭的一个框架用的5，之前没有出现过这个问题
